kwok/cloudprovider/cloudprovider.go

```diff
@@ -119,7 +120,13 @@ func (c CloudProvider) Name() string {
 }
 
 func (c CloudProvider) GetSupportedNodeClasses() []schema.GroupVersionKind {
-	return []schema.GroupVersionKind{}
+	return []schema.GroupVersionKind{
+		{
+			Group:   v1alpha1.SchemeGroupVersion.Group,
+			Version: v1alpha1.SchemeGroupVersion.Version,
+			Kind:    "KwokNodeClass",
```

Comment:
```suggestion
			Kind:    "KWOKNodeClass",
```
Given that we do `EC2NodeClass` and not `Ec2NodeClass`